warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownFunction.java

```diff
@@ -175,9 +173,13 @@ protected void redistributeQueries(Multimap<String,QueryPlan> serverPlan, Tablet
         while (true) {
             
             binnedRanges.clear();
-            AuthenticationToken authToken = new PasswordToken(config.getAccumuloPassword());
-            Credentials creds = new Credentials(config.getClient().whoami(), authToken);
-            List<Range> failures = tl.binRanges((ClientContext) client, ranges, binnedRanges);
+            ClientContext ctx;
+            if (client instanceof WrappedAccumuloClient) {
+                ctx = (ClientContext) ((WrappedAccumuloClient) client).getReal();
+            } else {
+                ctx = (ClientContext) client;
+            }
```

Comment:
I think there's a lot of this in the code. Would probably be a good idea to move this into a static utility method somewhere.

Reply:
Moved to static utility method  in latest commit